plugins/package-managers/node/src/main/kotlin/Npm.kt

```diff
@@ -382,9 +377,9 @@ open class Npm(
             }
         }
 
-    protected open fun getRemotePackageDetails(workingDir: File, packageName: String): JsonNode {
-        val process = run(workingDir, "info", "--json", packageName)
-        return jsonMapper.readTree(process.stdout)
+    internal open fun getRemotePackageDetails(workingDir: File, packageName: String): PackageJson {
```

Comment:
I believe this can stay `proetcted`, or?

Reply:
No it can't as long as `PackageJson` remains `internal`, because one must not expose `internal`s which `protected` here would do.

Reply:
Hmm, but it compiles at least with `protected` for me. Is that expected?

Reply:
Thanks for insisting. I was 100% sure I did this change in response to a compile issue, but it was a mistake.